### DISCUSSION TITLE
能不能出一个具体添加站点的教程

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [Goalonez/goalonez.github.io#168] [Q&A]
或者给一个可用站点的邀请码

**Comment by Goalonez** [comment]
> 或者给一个可用站点的邀请码


你指的是moviepolit那篇教程的站点吗？添加站点用cookiecloud插件，或者手动填写cookie之类的信息。站点的邀请码可以去贴吧“PT吧”看看，会有一些人发邀请。
